### ISSUE TITLE
Keyboard navigation between fields doesn't work

### ISSUE THREAD

**Issue by sashakavun** [shellscape/Gmail-Notifier-Plus#83]
Version:
3.0.0.0

OS:
Windows 2008 Server R2 SP1 x64

Steps to reproduce:
1. Open settings window
2. Try to use Tab/Shift+Tab to reach any of the controls
